yapi.py

```diff
@@ -28,16 +38,55 @@ def print_commands_allowed():
             print("\t - {} \n\t\t python yapi.py {}".format(
                 options[option][1], option))
 
-
 def argumentError(arg):
     """Argument error."""
     print("The argument {} isn't allowed,".format(arg.upper()))
     print_commands_allowed()
 
+def build(app):
```

Comment:
I have transfered this functionality on this file from your precedent PR -> [user_interface.py](https://github.com/Wabri/YAPI/blob/master/user_interface.py)